fe/fe-core/src/test/resources/sql/tpch-histogram-cost/histogram-stats/p_type.json

```diff
@@ -1 +1 @@
D COPPER","1327"],["LARGE PLATED NICKEL","1332"],["MEDIUM ANODIZED NICKEL","1331"],["STANDARD PLATED NICKEL","1336"],["LARGE BRUSHED COPPER","1333"],["ECONOMY BRUSHED NICKEL","1331"],["LARGE PLATED COPPER","1324"],["ECONOMY BRUSHED BRASS","1334"],["ECONOMY ANODIZED BRASS","1332"],["LARGE BURNISHED COPPER","1331"],["STANDARD ANODIZED COPPER","1327"],["ECONOMY BRUSHED STEEL","1329"],["MEDIUM POLISHED BRASS","1335"],["MEDIUM ANODIZED STEEL","1327"],["SMALL BRUSHED TIN","1328"],["SMALL BRUSHED NICKEL","1338"],["SMALL BURNISHED TIN","1335"],["LARGE POLISHED TIN","1331"],["MEDIUM BURNISHED BRASS","1330"],["STANDARD PLATED STEEL","1337"],["SMALL ANODIZED STEEL","1333"],["SMALL POLISHED BRASS","1335"],["MEDIUM BRUSHED STEEL","1331"],["STANDARD BRUSHED BRASS","1339"],["STANDARD POLISHED STEEL","1332"],["STANDARD ANODIZED TIN","1335"],["MEDIUM BRUSHED TIN","1332"],["SMALL ANODIZED TIN","1344"],["LARGE BURNISHED NICKEL","1336"],["LARGE BURNISHED STEEL","1331"],["PROMO ANODIZED NICKEL","1333"],["PROMO POLISHED COPPER","1333"],["MEDIUM POLISHED TIN","1338"],["PROMO BRUSHED BRASS","1325"],["LARGE BRUSHED BRASS","1327"],["STANDARD BURNISHED BRASS","1335"],["PROMO BURNISHED STEEL","1332"],["STANDARD BRUSHED COPPER","1334"],["MEDIUM ANODIZED COPPER","1335"],["STANDARD BURNISHED NICKEL","1335"],["STANDARD PLATED BRASS","1335"],["STANDARD POLISHED TIN","1329"],["SMALL PLATED COPPER","1334"],["SMALL BURNISHED COPPER","1335"],["SMALL POLISHED NICKEL","1328"],["STANDARD ANODIZED STEEL","1327"],["STANDARD POLISHED NICKEL","1336"],["STANDARD ANODIZED BRASS","1330"],["STANDARD ANODIZED NICKEL","1333"],["MEDIUM BURNISHED STEEL","1330"],["SMALL BRUSHED STEEL","1332"],["STANDARD POLISHED COPPER","1334"],["SMALL BRUSHED BRASS","1331"],["LARGE ANODIZED STEEL","1334"],["SMALL PLATED BRASS","1334"],["SMALL PLATED NICKEL","1335"],["MEDIUM BURNISHED NICKEL","1330"],["MEDIUM ANODIZED TIN","1336"],["PROMO BURNISHED COPPER","1333"],["LARGE ANODIZED COPPER","1333"],["MEDIUM BURNISHED TIN","1334"],["SMALL POLISHED STEEL","1333"],["SMALL PLATED STEEL","1336"],["LARGE BRUSHED NICKEL","1337"],["PROMO POLISHED STEEL","1332"],["PROMO BURNISHED BRASS","1331"],["PROMO PLATED TIN","1333"],["LARGE BRUSHED TIN","1334"],["MEDIUM ANODIZED BRASS","1334"],["PROMO BURNISHED NICKEL","1333"],["LARGE POLISHED BRASS","1334"],["ECONOMY PLATED COPPER","1331"],["PROMO PLATED BRASS","1332"],["STANDARD BURNISHED COPPER","1336"],["LARGE BRUSHED STEEL","1338"],["PROMO PLATED COPPER","1336"],["ECONOMY PLATED TIN","1334"],["PROMO POLISHED TIN","1334"],["ECONOMY POLISHED TIN","1334"],["ECONOMY POLISHED NICKEL","1333"],["ECONOMY PLATED STEEL","1332"],["ECONOMY BURNISHED STEEL","1330"],["ECONOMY POLISHED BRASS","1333"],["ECONOMY ANODIZED COPPER","1336"],["ECONOMY BURNISHED TIN","1333"],["ECONOMY ANODIZED STEEL","1335"],["ECONOMY BURNISHED NICKEL","1334"],["PROMO ANODIZED BRASS","1338"],["PROMO ANODIZED COPPER","1332"],["LARGE POLISHED NICKEL","1334"],["LARGE POLISHED STEEL","1334"],["SMALL BRUSHED COPPER","1336"]]}
```

Comment:
there is no buckets. we use mcv to supplement. Otherwise the plan will be wrong due to CE error.